<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPlaneExport_UI</class>
 <widget class="QDialog" name="ColorPlaneExport_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export color plane to PNG</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblSpace">
     <property name="text">
      <string>Color space</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="KComboBox" name="cbColorspace">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="lblVariant">
     <property name="text">
      <string>Variant</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="KComboBox" name="cbVariant"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblSliderName">
     <property name="text">
      <string comment="KDE::DoNotExtract">(notransl.)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSlider" name="sliderColor">
     <property name="whatsThis">
      <string/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="lblColNr">
     <property name="text">
      <string comment="KDE::DoNotExtract">(notranslate) Y display</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblScaling">
     <property name="text">
      <string>Scaling</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="sliderScaling">
     <property name="toolTip">
      <string>How much to zoom in</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="lblScaleNr">
     <property name="text">
      <string comment="KDE::DoNotExtract">(notranslate) % display</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblRes">
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="tResX">
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="lblTimes">
     <property name="text">
      <string notr="true">Ã—</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="tResY">
     <property name="maxLength">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QLabel" name="lblSize">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string comment="KDE::DoNotExtract">(notranslate) Total pixels go here</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblFilename">
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="3">
    <widget class="KUrlRequester" name="kurlrequester">
     <property name="mode">
      <set>KFile::File|KFile::LocalOnly</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorPlaneExport_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorPlaneExport_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
